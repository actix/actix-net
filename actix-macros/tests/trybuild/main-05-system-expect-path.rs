#[actix_rt::main(system = "!@#*&")]
async fn main2() {}

fn main() {}
