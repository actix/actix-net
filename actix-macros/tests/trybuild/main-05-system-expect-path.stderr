error: Expected path
 --> $DIR/main-05-system-expect-path.rs:1:27
  |
1 | #[actix_rt::main(system = "!@#*&")]
  |                           ^^^^^^^
